xml/templates/civicrm_state_province.tpl

```diff
@@ -4166,4 +4166,18 @@ INSERT INTO civicrm_state_province (id, country_id, abbreviation, name) VALUES
 (NULL, 1101, "LA", "Ladākh")
 
  -- end of statement:
+
+-- Add missing provinces for Luxembourg
+(NULL, 1020, 'CA', 'Capellen'),
+(NULL, 1020, 'CL', 'Clervaux'),
+(NULL, 1020, 'EC', 'Echternach'),
+(NULL, 1020, 'ES', 'Esch-sur-Alzette'),
+(NULL, 1020, 'GR', 'Grevenmacher'),
+(NULL, 1020, 'ME', 'Mersch'),
+(NULL, 1020, 'RD', 'Redange-sur-Attert'),
+(NULL, 1020, 'RM', 'Remich'),
+(NULL, 1020, 'VD', 'Vianden'),
+(NULL, 1020, 'WI', 'Wiltz'),
```

Comment:
This line shouldn't end with a comma, and you need to add a comma at line 4166.

Reply:
fixed